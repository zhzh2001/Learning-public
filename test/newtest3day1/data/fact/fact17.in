89477612 10