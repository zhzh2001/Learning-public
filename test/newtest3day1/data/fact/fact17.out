5.098544513e+672640785
